src/login/logind-dbus.c

```diff
@@ -2945,6 +2946,16 @@ int manager_start_scope(
                         return r;
         }
 
+        if (!isempty(requires)) {
+                r = sd_bus_message_append(m, "(sv)", "After", "as", 1, requires);
+                if (r < 0)
+                        return r;
+
+                r = sd_bus_message_append(m, "(sv)", "Requires", "as", 1, requires);
+                if (r < 0)
+                        return r;
+        }
```

Comment:
what's the rationale for using Requires= here, and not Wants= or BindsTo=? Can you elaborate?